packages/jest-cli/src/init/constants.ts

```diff
@@ -1,17 +0,0 @@
-/**
- * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
- *
- * This source code is licensed under the MIT license found in the
- * LICENSE file in the root directory of this source tree.
- */
-
-export const PACKAGE_JSON = 'package.json';
```

Comment:
this was essentially a duplicate of the file in `jest-config`, so I just exported it from there and deleted this